classes/publish/package/index.js

```diff
@@ -84,6 +84,7 @@ module.exports = class PublishApp {
             author: {},
             org: '',
             version: this.version,
+            response: {},
```

Comment:
This basically store the response from the server. When doing so I'm in doubt how much value some of the other properties on the outgoing object has. Is there any better way of doing this?

Reply:
The outgoing object is basically a piece of state that gets passed to a formatter once everything is complete so that we can print out nicely formatted info to the user.

Reply:
Right.